### DISCUSSION TITLE
Remove {{ addReactions ... }} calls in counting cc.

### DISCUSSION THREAD

**Discussion by ronaksahai** [yagpdb-cc/yagpdb-cc#253] [Q&A] [answered]
If a user has YAGPDB blocked, the CC will give an error and does not update the db as well.

**Comment by l-zeuch** [comment, marked as answer]
This was resolved in [`483f720`](https://github.com/yagpdb-cc/yagpdb-cc/commit/483f720873d444fc850905472f569286d9a7fb8a) via PR #303. Thanks for the report.
